apps/rich-list-api/src/modules/RootModule.ts

```diff
@@ -23,15 +28,20 @@ import { QueueItem, QueueModule } from './QueueModule'
         username: configService.get<string>('POSTGRES_USER'),
         password: configService.get<string>('POSTGRES_PASS'),
         database: configService.get<string>('POSTGRES_DB'),
-        migrations: [/* TODO(@ivan-zynesis): create table if not exists */],
-        // FIXME(@ivan-zynesis): remove sync after entities loading can be done via migration
-        entities: [QueueItem],
+        entities: [
```

Comment:
Tested with typeormmodule sync schema, it is not wiping any data upon service restart or reconnect.
To minimize code maintenance, we will go without any migration script. Any schema changes will let the data resync/reindex from genesis which is fine.